en/ui-system/components/editor/videoplayer.md

```diff
@@ -17,51 +17,51 @@ For more information about VideoPlayer's scripting interface, please refer to th
 ## VideoPlayer Properties
 
 | Properties | Function Explanation |
-|-------- | ----------- |
-| Resource Type        | The resource type of videoplayer, REMOTE for remote url and LOCAL for local file path. |
-| Remote URL           | Displayed when Resource Type is REMOTE, feed it with a remote video URL. |
-| Clip                 | Displayed when Resource Type is LOCAL, feed it with a local video path. |
-| Play On Awake        | Whether the video start playing automatically after loaded? |
-| Current Time         | The current playback time of the now playing item in seconds, you could also change the start playback time. |
-| Volume               | The volume of the video. (0.0 ~ 1.0) |
-| Mute                 | Mutes the VideoPlayer. Mute sets the volume=0, Un-Mute restore the original volume. |
-| Keep Aspect Ratio    | Whether keep the aspect ratio of the original video. |
-| Full Screen On Awake | Whether play video in fullscreen mode. |
-| Stay On Bottom       | Display video below the game view (Only available on web). |
-| Video Player Event   | The video player's callback, it will be triggered when certain event occurs. Please refer to the `VideoPlayer Event` section below or [VideoPlayerEvent API](../../../api/en/classes/video.videoplayer.html#videoplayerevent) for more details. |
-
-**Note:** In **Node** of the **Video Player Event** property, you should fill in a Node that hangs the user script component, and in the user script you can use the relevant VideoPlayer event according to the user's needs.
+|:-------- | :----------- |
+| **Resource Type**        | The resource type of videoplayer, REMOTE for remote url and LOCAL for local file path. |
+| **Remote URL**           | Displayed when Resource Type is REMOTE, feed it with a remote video URL. |
+| **Clip**                | Displayed when Resource Type is LOCAL, feed it with a local video path. |
+| **Play On Awake**        | Whether the video start playing automatically after loaded? |
+| **Current Time**         | The current playback time of the now playing item in seconds, you could also change the start playback time. |
+| **Volume**               | The volume of the video. (0.0 ~ 1.0) |
+| **Mute**                 | Mutes the VideoPlayer. Mute sets the volume=0, Un-Mute restore the original volume. |
+| **Keep Aspect Ratio**    | Whether keep the aspect ratio of the original video. |
+| **Full Screen On Awake** | Whether play video in fullscreen mode. |
+| **Stay On Bottom**       | Display video below the game view (Only available on web). |
+| **Video Player Event**   | The video player's callback, it will be triggered when certain event occurs. Please refer to the `VideoPlayer Event` section below or [VideoPlayerEvent API](../../../api/en/classes/VideoPlayer.html#videoplayerevent) for more details. |
```

Comment:
The original link is correct and does not need to be modified.

Reply:
I didn't change this, it happened when I did a `git pull` from the `v3.0` branch. See, they are incorrect in that branch, every link you mention: https://raw.githubusercontent.com/cocos-creator/creator-docs/v3.0/en/ui-system/components/editor/videoplayer.md

I fixed them now in `v3.0-release`, i.e. this PR.